source/contracts/reporting/Market.sol

```diff
@@ -144,7 +144,8 @@ contract Market is DelegationTarget, ITyped, Initializable, Ownable, IMarket {
     }
 
     function contribute(uint256[] _payoutNumerators, bool _invalid, uint256 _amount) public returns (bool) {
-        require(feeWindow.isActive());
+        require(getInitialReporter().getReportTimestamp() != 0);
```

Comment:
I believe this line has no functional effect. A market is not assigned a FeeWindow until it is InitialReport'd on (open or designated). This check will pass any time after initial report (when there is a non-zero FeeWindow). Calling isActive on FeeWindow(0) would also fail.

This is more explicit, and better behavior than making a call to FeeWindow(0)